docs/api.md

```diff
@@ -1805,6 +1805,7 @@ await page.tracing.stop();
 - `options` <[Object]>
   - `path` <[string]> A path to write the trace file to.
   - `screenshots` <[boolean]> captures screenshots in the trace.
+  - `jssampling` <[boolean]> enable js sampling. Defaults to `false`.
```

Comment:
What does this do?

Reply:
ive always thought calling this "sampling" is really unhelpful to end users. same with "js samples", though that'd be more accurate.

those names do kinda help set expectations for people who will consume with this data, as working w/ sampling data is a bit more complicated than working with tracing data. but that's a whole 'nother thing.  

other potential names:

* `callstacks`
* `jsCallstacks`
* `javascript`